.github/workflows/canary.yaml

```diff
@@ -67,6 +67,27 @@ jobs:
       - name: Test
         run: wing test
         working-directory: checks
+  canary-eventbridge:
+    name: Test eventbridge
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
hmm 🤔 , why was this added?

Reply:
@MarkMcCulloh / @Chriscbr ? 

Do you guys know how was this added to my PR? it seems to be created by me running `./mklib.sh simtools`

Reply:
Seems like a drift between the script that generates these workflows and `main`. 

@Chriscbr we should add some sort of anti-tamper or self mutation to this repo.